setup.py

```diff
@@ -29,11 +31,15 @@
     ],
     keywords='dataset data iteration pipeline processing',
     packages=find_packages(exclude=['tests']),
-    install_requires=['six', 'picklable_itertools', 'pyyaml', 'h5py',
+    install_requires=['six', 'picklable_itertools', 'pyyaml', 'h5py', 'cython',
                       'tables', 'progressbar2', 'pyzmq', 'scipy', 'pillow',
                       'requests'],
     extras_require={
         'test': ['nose', 'nose2', 'mock']
     },
-    scripts=['bin/fuel-convert', 'bin/fuel-download', 'bin/fuel-info']
+    scripts=['bin/fuel-convert', 'bin/fuel-download', 'bin/fuel-info'],
+    ext_modules=cythonize(Extension("fuel.transformers._image",
```

Comment:
Pinged @bartvm a little earlier, he didn't have a problem with it. It'll automatically get done on `python setup.py install`.